src/components/Thumbs.js

```diff
@@ -304,7 +304,7 @@ class Thumbs extends Component {
         return (
             <div className={klass.CAROUSEL(false)}>
                 <div className={klass.WRAPPER(false)} ref={this.setItemsWrapperRef}>
-                    <button type="button" className={klass.ARROW_PREV(!hasPrev)} onClick={this.slideRight} />
+                    <button type="button" className={klass.ARROW_PREV(!hasPrev)} onClick={this.slideRight} role="presentation" aria-label="mobile navigation"/>
```

Comment:
Should the `aria-label` be something like 'next slide / item' and 'previous slide / item'? 

Also, I suggest to make these strings be props. Any string used to communicate with end users need to be translated and using props is a good way to do so.

Reply:
@leandrowd Totally agree with your statement as a end-user. Good default such as "previous"/"next" is also a good thing.

I'm not sure if the `role=presentation` is really needed, see https://dequeuniversity.com/rules/axe/2.2/button-name?application=lighthouse

Reply:
@Vadorequest I'm sorry, I maybe reading this wrong but the 4th bullet point says: `role="presentation" or role="none" (ARIA 1.1) and is not in tab order (tabindex="-1").` 
Isn't that saying that a button needs a role?

P.S Sorry it took me so long to get back to this PR, life and stuff...

Reply:
@leandrooriente  Do you think there should be two props like `ariaLabelLeft` and ariaLabelRight` or one object with both in it like:
```
<Carousel ariaLable={['crazy prev slide button', `cool next slide button`]} >
```
or 
```
<Carousel ariaLable={{prev: "super cool prev button", next: "the alright next button"}} >
```
and defaults are  `next slide / item` and `previous slide / item` per your suggestion.

Reply:
Wrong Leandro dude. 

cc: @leandrowd